DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -315,10 +318,11 @@ public void Configure(IApplicationBuilder app, IMigrationRunner migrationRunner,
 
         private Task RedirectToLogin(RedirectContext<CookieAuthenticationOptions> context)
         {
-            var applicationPath = new Uri(config.GetAppRootPath()).AbsolutePath.TrimEnd('/');
+            // TODO HEEDLS-609 update this
+            var applicationPath = new Uri(ConfigHelper.GetAppRootPath(config)).AbsolutePath.TrimEnd('/');
             var url = HttpUtility.UrlEncode(applicationPath + context.Request.Path);
             var queryString = HttpUtility.UrlEncode(context.Request.QueryString.Value);
-            context.HttpContext.Response.Redirect(config.GetAppRootPath() + $"/Login?returnUrl={url}{queryString}");
+            context.HttpContext.Response.Redirect(ConfigHelper.GetAppRootPath(config) + $"/Login?returnUrl={url}{queryString}");

```

Comment:
I'd say just port this helper method over to the Data project in this commit. I'm guessing VS can move static methods for you with its refactoring tool? Rider can.

Reply:
The method is already in the data project it turns out, I just need to point the code at the right one (this whole original refactor was done automatically and I didn't notice it at first, an example of the IDE being too smart).

Reply:
We'll leave this as it was.